packages/bruno-lang/v2/src/bruToJson.js

```diff
@@ -374,11 +374,14 @@ const sem = grammar.createSemantics().addAttribute('ast', {
   authbearer(_1, dictionary) {
     const auth = mapPairListToKeyValPairs(dictionary.ast, false);
     const tokenKey = _.find(auth, { name: 'token' });
+    const prefixKey = _.find(auth, { name: 'prefix' });
     const token = tokenKey ? tokenKey.value : '';
+    const prefix = prefixKey ? prefixKey.value : '';
```

Comment:
I think this should be 
`const prefix = prefixKey ? prefixKey.value : 'Bearer';`

The other option would be to update the logic in prepare request code to use the prefix as `Bearer` if the prefix is empty.